doc/libpmemobj/pmemobj_tx_begin.3.md

```diff
@@ -315,6 +320,68 @@ The **TX_END** macro cleans up and closes the transaction started by the
 It is mandatory to terminate each transaction with this macro. If the transaction
 was aborted, *errno* is set appropriately.
 
+## TUNING ##
+
+From pmemobj implementation perspective there are two types of operations
+in a transaction:
+
++ **snapshots**, where action must be performed and persisted immediately
++ **intents**, where the persist part of the action can be postponed until
+  transaction commit
+
+**pmemobj_tx_add_range**() and all its variants belong to the **snapshots**
+group.
+
+**pmemobj_tx_alloc**() (with its variants), **pmemobj_tx_free**(),
```

Comment:
I am not sure, you need to check it in other mans:
**pmemobj_tx_alloc**() -> **pmemobj_tx_alloc**(3)